integration-tests/src/cross_chain_transfer.rs

```diff
@@ -110,7 +110,7 @@ fn rococo_should_receive_asset_when_sent_from_hydra() {
 	Rococo::execute_with(|| {
 		assert_eq!(
 			hydradx_runtime::Balances::free_balance(AccountId::from(BOB)),
-			2_999_989_698_923 // 3 * HDX - fee
+			2_999_989_606_492 // 3 * HDX - fee
```

Comment:
Why did the fee change?

Reply:
It is coming from using `WirthUniqueTopic` for Router